app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/logo"
+        android:layout_width="@dimen/logo_width"
+        android:layout_height="@dimen/logo_height"
+        android:background="@drawable/ic_vector"
+        android:contentDescription="@string/ic_logo_title"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/logoCaption"
```

Comment:
This is not needed. Generally we only need to give 1 horizontal and 1 vertical constraint. In this case 2 horizontal because we want the image in the middle. So we can remove the layout_constraintBottom_toTopOf